datalad/core/local/save.py

```diff
@@ -322,3 +334,40 @@ def __call__(path=None, message=None, dataset=None,
                     status='error',
                     message=('cannot tag this version: %s', e.stderr.strip()))
                 yield dsres
+
+
+def get_commit_message(repo, recursive):
+    import os
+    from ...utils import edit_text_in_file
+    msg_file = repo.pathobj / '.git' / 'COMMIT_EDITMSG'
```

Comment:
probably we shouldn't mess with files used by git itslf, and name this one `SAVE_EDITMSG`